src/hdf5_interface.F90

```diff
@@ -2347,6 +2426,67 @@ subroutine read_attribute_string(buffer, obj_id, name)
     call h5tclose_f(memtype, hdf5_err)
   end subroutine read_attribute_string
 
+  subroutine read_attribute_string_1D(buffer, obj_id, name)
+    character(*), target, allocatable, intent(inout) :: buffer(:)
+    integer(HID_T), intent(in) :: obj_id
+    character(*),   intent(in) :: name
+
+    integer          :: hdf5_err
+    integer(HID_T)   :: space_id
+    integer(HID_T)   :: attr_id
+    integer(HSIZE_T) :: dims(1)
+    integer(HSIZE_T) :: maxdims(1)
+
+    call h5aopen_f(obj_id, trim(name), attr_id, hdf5_err)
+
+    if (allocated(buffer)) then
+      dims(:) = shape(buffer)
+    else
+      call h5aget_space_f(attr_id, space_id, hdf5_err)
+      call h5sget_simple_extent_dims_f(space_id, dims, maxdims, hdf5_err)
```

Comment:
Ahh there's something tricky with the Python2/Python3 thing here.  I had ruled that out by running `python3 get_nndc_data.py`.  But `get_nndc_data.py` simply calls `openmc-ace-to-hdf5` which uses just `python` aka Python2 on my system.  Glad I haven't gotten around to aliasing `python` to `python3` yet!